Lib/test/test_c_locale_coercion.py

```diff
@@ -134,10 +134,12 @@ def _handle_output_variations(data):
 
         * Some platforms report ASCII as ANSI_X3.4-1968
         * Some platforms report ASCII as US-ASCII
+        * Some platforms report ASCII as 646
         * Some platforms report UTF-8 instead of utf-8
         """
         data = data.replace(b"ANSI_X3.4-1968", b"ascii")
         data = data.replace(b"US-ASCII", b"ascii")
+        data = data.replace(b"646", b"ascii")
```

Comment:
There is a alias iso_646. Would not it be convenient to add it?

Reply:
Well, it can be done, but there are some many ASCII aliases; maybe its reasonable to add only those that are actually used by platforms.